script.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+
+        var letterGrade = $('#letter-grade').val().toUpperCase();
+        var myFavoriteThings = [];
+            myFavoriteThings.push("My First favorite thing is faith");   
+            myFavoriteThings.push("My Second favorite thing is friends"); 
+            myFavoriteThings.push("My Third favorite thing is family"); 
+            myFavoriteThings.push("My Fourth favorite thing is running"); 
+
+        //Write your decision structure between this comment...
+if (letterGrade == A){alert("Grade range for A: 90% to 100%");}
+if (letterGrade == B){alert("Grade range for B: 80% to 90%");}
+if (letterGrade == C){alert("Grade range for C: 70% to 80%");}
+if (letterGrade == D){alert("Grade range for D: 60% to 70%");}
+if (letterGrade == F){alert("Grade range for F: Less than 60%");}
+else (letterGrade ==){alert("Error. Invalid value entered.");}
```

Comment:
I would prefer to see and if/else/if structure here with the text formatting shown in the powerpoint. Additionally - you are not updating the text of the html elements as requeired. Something like:
if (lettergrade=='A') {
    $('#decision-holder').text("Grade range for A: 90% to 100%");
} else if (lettergrade=='B') {
...

Let me know if you have questions.